.gitmodules

```diff
@@ -1,3 +1,7 @@
-[submodule "MinHook"]
```

Comment:
Can you check submodules pull correctly? Moving them is a little pain, because there is no move, only remove and readd...

Reply:
Seems to be working fine